app/src/main/java/practicum/getfitla_v3/Nutrition.java

```diff
@@ -1,86 +1,210 @@
 package practicum.getfitla_v3;
 
+import android.content.Intent;
 import android.os.Bundle;
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.widget.LinearLayoutManager;
 import android.support.v7.widget.RecyclerView;
 import android.view.View;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
 import java.util.ArrayList;
 import java.util.List;
-import android.content.Intent;
-import android.content.Context;
-
 
 /**
  * An activity that displays a the list of Recipe Items
  */
-public class Nutrition extends AppCompatActivity implements ItemClickListener{
+public class Nutrition extends AppCompatActivity implements ItemClickListener {
+
+    static final String EXERCISE_SHEET_URL = "https://spreadsheets.google.com/tq?key=1a6YXr5Tf_nREmb9byNWLLtpKZEfIUPnDq1JLsaMXWig";
+
 
     //Creating instance of recyclerview
     RecyclerView recyclerView;
     //Creating a new list that will take object of type NutritionItemFormat, or specifically items that have been formatted
-    List<NutritionItemFormat> nutritionList;
+    public List<NutritionItemFormat> nutritionList = new ArrayList<>();
     //Initializing my Adaptor
     private NutritionListAdapter mAdapter;
-    private Context mContext;
+
+    private void processJson(JSONObject object) {
```

Comment:
So you can put both the JSON process/object creation together with the code that you actually render the view with, but it would be cleaner to have one class where you read in all this JSON shit and write a method that throws it all in an ArrayList, and then have  separate class to do onCreate